### PR TITLE
Add email subscription form

### PR DESCRIPTION
**Pull request by luise**
Now: 
screen shot 2017-11-27 at 8 10 23 pm

Before:
screen shot 2017-11-27 at 5 48 11 pm

### CODE REVIEW

**Comment by luise** [comment]
@kayousterhout , there is no rush getting this reviewed. It is last priority for the sprint.

File: index.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
 		<meta name="viewport" content="initial-scale=1, maximum-scale=1">
 		<meta name="author" content="Kelda">
 		<meta name="description" content="Kelda">
+		<link rel="stylesheet" href="https://cdn-images.mailchimp.com/embedcode/horizontal-slim-10_7.css" type="text/css">
```
Comment (kklin) [inline index.html]: Is this style sheet meant to be referenced by external users? Or is it just an example, and we should copy the relevant stuff into our CSS?

**Comment by kayousterhout** [comment]
Can we just say "...send you the latest news about Kelda"? The "very occasionally" is perfect (and I think "biggest and latest" sounds a little overkill)

File: index.html (hunk 1 above)
Comment (luise) [reply index.html]: It's meant to be referenced by users directly -- they have a snippet they suggest copy pasting, and this is part of it.

**Comment by luise** [comment]
@kayousterhout , I updated it. And thanks for the double review, @kklin 😄

**Pull request merged**